pkg/users/group-details.js

```diff
@@ -0,0 +1,218 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import React, { useState } from 'react';
+import { superuser } from "superuser";
+import { apply_modal_dialog } from "cockpit-components-dialog.jsx";
+import { delete_group_dialog } from "./delete-group-dialog.js";
+import { show_unexpected_error } from "./dialog-utils.js";
+
+import {
+    Button,
+    Card, CardBody, CardHeader, CardTitle, CardActions,
+    Flex, HelperText, HelperTextItem,
+    Label, LabelGroup,
+    Page, PageSection,
+    Select, SelectOption, SelectVariant, Text, TextVariants, Breadcrumb, BreadcrumbItem,
+    Form, FormGroup, TextInput,
+} from '@patternfly/react-core';
+import { UndoIcon } from '@patternfly/react-icons';
+
+const _ = cockpit.gettext;
+
+export function GroupDetails({ groups, accounts, group }) {
+    const [edited_name, set_edited_name] = useState(null);
+
+    function change_name() {
+        if (!edited_name)
+            return;
+
+        // TODO: unwanted chars check
+        cockpit.spawn(["/usr/sbin/groupmod", group, "--new-name", edited_name],
+                      { superuser: "try", err: "message" })
+                .then(() => {
+                    set_edited_name(null);
+                    cockpit.location.go("#/group/" + edited_name);
+                })
+                .catch(error => {
+                    set_edited_name(null);
+                    show_unexpected_error(error);
+                });
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230109-123701-1958283d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)